stories/features/reboot.fmf

```diff
@@ -0,0 +1,31 @@
+title: Reboot during test
+story:
+    As a tester I need to reboot the guest during a test and then
+    resume the test execution.
+description: |
+    Some tests may require a reboot as a part of them, e.g.
+    upgrading the system, rebooting and then running some checks.
+
+    The functionality is compatible with how restraint
+    works, hence ``rhts-reboot``, ``rstrnt-reboot`` or ``tmt-reboot``
+    can be used to start the reboot. The ``REBOOT_COUNT`` environment
```

Comment:
Calling the variable `REBOOT_COUNT` seems to contradict "The functionality is compatible with how restraint works", because restraint does not have such variable (at least not documented). According to https://restraint.readthedocs.io/en/latest/remove_rhts.html?highlight=rebootcount#legacy-rhts-task-environment-variables it has `REBOOTCOUNT` (note the missing underscore) and `RSTRNT_REBOOTCOUNT` .